src/Async/index.js

```diff
@@ -131,7 +131,7 @@ function Async(fn) {
 
   function fork(reject, resolve, cleanup) {
     if(!isFunction(reject) || !isFunction(resolve)) {
-      throw new TypeError('Async.fork: Reject and resolve functions required')
+      throw new TypeError('Async.fork: Resolve and reject arguments must both be Functions')
```

Comment:
What is the purpose in switching the order here. I can see that originally it was Reject then Resolve in the word order because that matches the order of the arguments

Reply:
no preference, IIRC my original issue was SVO phrasing

Reply:
Lets go ahead and change the order, if you like i'm happy to make the change myself then we can merge this in

Reply:
Yes, please, and thank you :)